docs/integrations/facebook/oauth.mdx

```diff
@@ -35,15 +38,23 @@ For additional assistance, see the [Facebook app registration](https://developer
 15. Select the toggle for "In development" at the top of the page and confirm switching to live mode.
 16. Your application should now show as live: ![](/img/howto/oauth/6-facebook-live.png)
 
-### Update your endpoint configuration
+### Update your ngrok Endpoint Traffic Policy
+
+1. Access the [ngrok Dashboard Endpoints page](https://dashboard.ngrok.com/endpoints?sortBy=createdAt&orderBy=desc) and locate an existing endpoint you'd like to add this to or create a new one.
+2. In your traffic policy, add the following configuration:
+
+<Facebook />
+
+A Note about Scopes: You may add [any scopes](https://developers.facebook.com/docs/apps/review/login-permissions) that are required by your application with the following caveats.
+
+- Scopes which require a Facebook [app review](https://developers.facebook.com/docs/apps/review/#app-review) are unsupported.
+- ngrok will enforce that users [accept all permissions](https://developers.facebook.com/docs/facebook-login/handling-declined-permissions#reprompt) before completing authorization.
+
+3. Click **Save** to validate and update your traffic policy.
+
+4. Optionally, configure access control to your service by only allowing specific users or domains.
```

Comment:
```suggestion
Optionally, configure access control to your service by only allowing specific users or domains.
```
Taking the number out because this isn't a step since it's optional